.github/workflows/pr.yml

```
@@ -0,0 +1,140 @@
+  workflow_dispatch:
+ # release:
+   # types: [published]
+
+jobs:
+  build-luxmeter:
+    name: Build firmware Luxmeter 🛠️
+    uses: esphome/workflows/.github/workflows/build.yml@main
+    with:
+      files: |
+        esphome/luxmeter.yaml
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)